### ISSUE TITLE
None: MoreByDeveloper 

### ISSUE THREAD

**Issue by hackrx** [JoMingyu/google-play-scraper#117]
**google_play_scraper.VERSION**
Write result of `google_play_scraper.VERSION`
1.0.3
**Describe the bug**
`moreByDeveloper` is None for apps which have `store/apps/dev?id=` in the link of developer details.
eg. for. `com.rhmsoft.code` the developer link is: `/store/apps/dev?id=5848068795804361091` and scraper returns `None` for it: 
but actually it have 7 more apps, so it should return all of these 

some more egs: `in.krosbits.musicolet` , `com.simplemobiletools.musicplayer`
Though it is working for app which have  `developer?id=` not `dev?id=` in the developer url, like `com.goodtoolapps.mp3playe`
developer link: `/store/apps/developer?id=GoodToolApps.com`
output:
```
['com.goodtoolapps.mp3converter']
```

**Code**

```
try:
  # 'com.rhmsoft.code'
    result = app(
        `in.krosbits.musicolet`,
    )    
    print(result['moreByDeveloper'])
except Exception as err:    
    print('error : ', err)

```

**Expected behavior**
Output should be:
```
['in.krosbits.castplugin']
```

**Comment by JoMingyu** [comment]
I'll handle it.

**Comment by hackrx** [comment]
Thanks @JoMingyu  for looking into this issue, any updates on this bug?

**Comment by JoMingyu** [comment]
@hackrx  ah, sorry. I was busy with my main job and missed it. I will release the update by fixing it up to April 30th.

**Comment by hackrx** [comment]
thanks @JoMingyu 🙂

**Comment by JoMingyu** [comment]
It fixed at version 1.0.5.

## as-is
```
result = app(
    'in.krosbits.musicolet',
)
print(result['moreByDeveloper'])
```

It returned `None`.

## now

```
result = app(
    'in.krosbits.musicolet',
)
print(result['moreByDeveloper'])
```

It returns `['in.krosbits.castplugin']`

**Comment by hackrx** [comment]
🎉🎉Thanks @JoMingyu, I tested the latest version and it is working as expected!
